tensorflow-framework/src/main/java/org/tensorflow/framework/initializers/RandomUniform.java

```diff
@@ -32,13 +32,12 @@
  *     RandomUniform&lt;TFloat32, TFloat32&gt; initializer =
  *              new org.tensorflow.framework.initializers.RandomUniform&lt;&gt;(tf, seed);
  *     Operand&lt;TFloat32&gt; values =
- *              initializer.call(tf.constant(Shape.of(2,2)), TFloat32.DTYPE);
+ *              initializer.call(tf.constant(Shape.of(2,2)), TFloat32.class);
  * </pre>
  *
  * @param <T> The TType for the call operation
- * @param <U> The TNumber for the call operation
  */
-public class RandomUniform<T extends TType, U extends TNumber> extends BaseInitializer<T> {
+public class RandomUniform<T extends TNumber> extends BaseInitializer<T> {
```

Comment:
`TFloating`?

Reply:
That one has the following check, so I assume that integral values are also accepted:
```java
    if (!dtype.isNumeric())
      throw new IllegalArgumentException("The data type must be numeric. Found : " + dtype.name());
```

Reply:
RandomNormal should be TFloating.

Reply:
[According to](https://github.com/tensorflow/java/pull/174#discussion_r547550405) @JimClarke5 , this one should remain valid for any numeral type.